ide/services/ide-editor/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I was thinking that in the future this new module could be used to provide backend support for non-core features that enhance and relate specifically to the editor like different custom formatter support (not only prettier), linting tools, ai hints, etc...
Correct me if i'm wrong, but "core" didn't sound right to me for features like this?

Reply:
The idea was that this module would hold functionality directly related to the Monaco editor and the stuff we load into it. Although with this PR the REST service exposes only an endpoint for fetching Prettier configs, there is another PR being prepared for having the ESLint run in the Monaco editor and it would also need to have a separate config file fetched. That would probably be implemented in this REST service as well. The `.d.ts` fetching would also be moved to this REST service at some point. Having said that, all of these features are tightly coupled with the Monaco editor and they are not much related to the IDE's core itself. Also, I think having a separate REST service for the Monaco editor and it being in a separate Maven module would be easier to maintain and extend in the future as it would be pretty clear this module holds functionality directly related to the Monaco editor